make/dev.mk

```diff
@@ -10,7 +10,68 @@ IMAGE_NAME?=member-operator
 TIMESTAMP:=$(shell date +%s)
 TAG?=$(GIT_COMMIT_ID_SHORT)-$(TIMESTAMP)
 
-# ToDo: add required make target to build and run operator in dev environment later once we have openshift-ci in place.
+# to watch all namespaces, keep namespace empty
+APP_NAMESPACE ?= ""
+LOCAL_TEST_NAMESPACE ?= "test-saas-control-plane"
 
+.PHONY: local
+## Run Operator locally
+local: login-as-admin create-namespace deploy-rbac build deploy-crd
+	$(Q)-oc new-project $(LOCAL_TEST_NAMESPACE) || true
+	$(Q)OPERATOR_NAMESPACE=$(LOCAL_TEST_NAMESPACE) operator-sdk up local --namespace=$(APP_NAMESPACE) --verbose
```

Comment:
It looks like `operator-sdk` is using `"-mod=vendor"`. So, we have to call `make vendor` here too:
```
Running []string{"go", "build", "-o", "build/_output/bin/member-operator-local", "-mod=vendor", "github.com/codeready-toolchain/member-operator/cmd/manager"}
```
Note: `make vendor` doesn't not exist in this repo yet. So we would need to copy it from https://github.com/codeready-toolchain/api/blob/8dbd7410a0532bcb4e6bcf2756460d37c7252e9a/make/go.mk#L22

Reply:
Correct, I had it already generated locally, so I didn't notice that it is needed :-/ 65573cd